From e2bb6424194b86ad5bb129e2925ff83b9b2261ae Mon Sep 17 00:00:00 2001
From: Haijun Zhang <haijun.zhang@freescale.com>
Date: Thu, 11 Apr 2013 11:53:42 +0800
Subject: [PATCH 366/547] eSDHC: remove duplicate code of sdhci-pltfm.c

Remove property of "broken-cd" from sdhci_get_of_property,
In current dts, there is no such property.
eSDHC had default quirk of SDHCI_QUIRK_BROKEN_CARD_DETECTION

Signed-off-by: Haijun Zhang <Haijun.Zhang@freescale.com>
CC: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
CC: Xie Xiaobo-R63061 <X.Xie@freescale.com>
Change-Id: I8d5bbd12e8ee0191978da25ea9bfa6482ddcf9b3
Reviewed-on: http://git.am.freescale.net:8181/1219
Reviewed-by: Xie Xiaobo-R63061 <X.Xie@freescale.com>
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
[Original patch taken from QorIQ-SDK-V1.4-SOURCE-20130625-yocto.iso]
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 drivers/mmc/host/sdhci-pltfm.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/mmc/host/sdhci-pltfm.c b/drivers/mmc/host/sdhci-pltfm.c
index 80823bd..c982cc3 100644
--- a/drivers/mmc/host/sdhci-pltfm.c
+++ b/drivers/mmc/host/sdhci-pltfm.c
@@ -83,9 +83,6 @@ void sdhci_get_of_property(struct platform_device *pdev)
 		if (sdhci_of_wp_inverted(np))
 			host->quirks |= SDHCI_QUIRK_INVERTED_WRITE_PROTECT;
 
-		if (of_get_property(np, "broken-cd", NULL))
-			host->quirks |= SDHCI_QUIRK_BROKEN_CARD_DETECTION;
-
 		if (of_get_property(np, "no-1-8-v", NULL))
 			host->quirks2 |= SDHCI_QUIRK2_NO_1_8_V;
 
-- 
1.8.4.93.g57e4c17

